### PR TITLE
Set up 1.14 CI (#11203 + patches + rebump)

### PR DESCRIPTION
**Pull request by krzyzacy**
/assign

So since @amwat had two bumps in his branch... what I did:

```
# Move back beyond the previous conflict bump
git reset HEAD~30 --hard

# checkout the PR
git checkout -b amwat-remove-1.10 master
git pull https://github.com/amwat/test-infra.git remove-1.10

# try to bump to previous 
(hard-coded TAG=`v20190214-f4092ae69`)
./experiment/bump_e2e_image.sh

# sync HEAD
git rebase upstream/master

# bump again
./experiment/bump_e2e_image.sh
```

and I also made a few changes on top of the original PR to fix the wrong image mappings

cc @imkin @BenTheElder @spiffxp

### CODE REVIEW

File: config/jobs/kubeflow/kubeflow-postsubmits.yaml [hunk 1]
```diff
@@ -179,7 +179,7 @@ postsubmits:
       preset-service-account: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-1.10
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-1.10
```
Comment (imkin) [inline config/jobs/kubeflow/kubeflow-postsubmits.yaml]: Don't we need to get rid/rotate the 1.10 configs?

File: config/jobs/kubernetes-security/generated-security-jobs.yaml [hunk 2]
```diff
@@ -1001,7 +1001,7 @@ presubmits:
         - --cluster=
         - --extract=local
         - --gcp-node-image=gci
-        - --gcp-zone=us-west1-b
+        - --gcp-zone=us-central1-f
```
Comment (imkin) [inline config/jobs/kubernetes-security/generated-security-jobs.yaml]: west1-b ?

File: config/jobs/kubernetes-security/generated-security-jobs.yaml [hunk 3]
```diff
@@ -1555,51 +1555,6 @@ presubmits:
     name: pull-security-kubernetes-cross
     rerun_command: /test pull-security-kubernetes-cross
     run_if_changed: ^((build\/|hack\/lib\/).*)|(.*Makefile.*)$
-    skip_branches:
```
Comment (imkin) [inline config/jobs/kubernetes-security/generated-security-jobs.yaml]: This should not have been part of release cut change. It gets difficult to automate such special changes. Also difficult to track why they were removed and not rotated to a new version.

File: config/jobs/kubernetes-security/generated-security-jobs.yaml [hunk 4]
```diff
@@ -3244,7 +3199,57 @@ presubmits:
           value: /workspace/k8s.io/kubernetes
         - name: METADATA_BANDAID
           value: "true"
-        image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-master
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-master
```
Comment (imkin) [inline config/jobs/kubernetes-security/generated-security-jobs.yaml]: Would it make sense to split the new job additions into another commit/PR so that they can be tracked?
Maybe we can add Comment/description?

File: config/jobs/kubernetes-security/generated-security-jobs.yaml [hunk 5]
```diff
@@ -3267,7 +3272,6 @@ presubmits:
     - release-1.13
```
Comment (imkin) [inline config/jobs/kubernetes-security/generated-security-jobs.yaml]: missing release-1.14

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml [hunk 6]
```diff
@@ -13,6 +13,6 @@ periodics:
     path_alias: "sigs.k8s.io/cluster-api-provider-aws"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-1.12
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-1.12
```
Comment (imkin) [inline config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml]: rotation missing.

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml [hunk 7]
```diff
@@ -7,7 +7,7 @@ presubmits:
     path_alias: "sigs.k8s.io/cluster-api-provider-aws"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-1.13
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-1.13
```
Comment (imkin) [inline config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml]: rotation missing (in the entire script). This job should move to 1.14

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml (hunk 6 above)
Comment (imkin) [reply config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml]: Pattern of rotation missing very common in this PR.

File: config/jobs/kubeflow/kubeflow-postsubmits.yaml (hunk 1 above)
Comment (krzyzacy) [reply config/jobs/kubeflow/kubeflow-postsubmits.yaml]: don't care about kubeflow

File: config/jobs/kubernetes-security/generated-security-jobs.yaml (hunk 2 above)
Comment (krzyzacy) [reply config/jobs/kubernetes-security/generated-security-jobs.yaml]: ack

File: config/jobs/kubernetes-security/generated-security-jobs.yaml (hunk 4 above)
Comment (krzyzacy) [reply config/jobs/kubernetes-security/generated-security-jobs.yaml]: this file/config is completely generated from other files, we want to avoid handcraft it.

File: config/jobs/kubernetes-security/generated-security-jobs.yaml (hunk 5 above)
Comment (krzyzacy) [reply config/jobs/kubernetes-security/generated-security-jobs.yaml]: 1.14 exists above

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml (hunk 7 above)
Comment (krzyzacy) [reply config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml]: this is presubmit for other repos, we should avoid touching their versions, aka a go version bump might break their repo

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml (hunk 6 above)
Comment (krzyzacy) [reply config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml]: no we don't care about cluster api

File: config/jobs/kubernetes-security/generated-security-jobs.yaml (hunk 2 above)
Comment (krzyzacy) [reply config/jobs/kubernetes-security/generated-security-jobs.yaml]: done (though it doesn't really matter anymore)

File: config/jobs/kubernetes/sig-gcp/gce-conformance.yaml [hunk 8]
```diff
@@ -103,11 +82,10 @@ periodics:
       - --provider=gce
       - --test_args=--ginkgo.focus=\[Conformance\] --ginkgo.skip=Alpha|\[(Disruptive|Feature:[^\]]+|Flaky)\]
       - --timeout=200m
-      image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-1.13
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-1.13
 
-# manual-release-bump-required (add a job for the new version and delete the job for the oldest version)
 - interval: 6h
-  name: ci-kubernetes-gce-conformance-stable-1-10
+  name: ci-kubernetes-gce-conformance-latest-1-14
```
Comment (imkin) [inline config/jobs/kubernetes/sig-gcp/gce-conformance.yaml]: seems like  
ci-kubernetes-gce-conformance-latest-1-14 and ci-kubernetes-gce-conformance-stable-1-14 job configs have merged.

File: config/jobs/kubernetes/kops/kops-config.yaml [hunk 9]
```diff
@@ -160,7 +160,7 @@ presubmits:
         - --aws-cluster-domain=test-cncf-aws.k8s.io
         - --check-leaked-resources=false
         - --cluster=
-        - --extract=ci/k8s-stable2
+        - --extract=ci/k8s-stable1
```
Comment (spiffxp) [inline config/jobs/kubernetes/kops/kops-config.yaml]: I'm guessing this is related to the `# manual-release-bump-required` comment above, but I don't understand why / what's going on here.  The job name references `stable2` and it's using the `release-1.12` branch, but this is using `stable1` which corresponds to `release-1.13`?

File: config/jobs/kubernetes/kops/kops-config.yaml [hunk 10]
```diff
@@ -199,7 +199,7 @@ presubmits:
         - --aws-cluster-domain=test-cncf-aws.k8s.io
         - --check-leaked-resources=false
         - --cluster=
-        - --extract=ci/k8s-stable3
+        - --extract=ci/k8s-stable2
```
Comment (spiffxp) [inline config/jobs/kubernetes/kops/kops-config.yaml]: same here, but `stable3` in name vs. `stable2` in use

File: config/jobs/kubernetes/sig-release/kubernetes-builds.yaml [hunk 11]
```diff
@@ -165,23 +133,23 @@ periodics:
...
-- interval: 1h
+- interval: 30m
   name: ci-kubernetes-build-stable2
   labels:
     preset-service-account: "true"
     preset-dind-enabled: "true"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/bootstrap:v20190211-fab6034b4
+    - image: gcr.io/k8s-testimages/bootstrap:v20190213-2f4f06285
```
Comment (imkin) [inline config/jobs/kubernetes/sig-release/kubernetes-builds.yaml]: should not we version the bootstrap image per release?

File: config/jobs/kubernetes/sig-aws/eks/eks-periodics.yaml [hunk 12]
```diff
@@ -73,53 +73,3 @@ periodics:
       - --eks-nodes=100
       - --test_args=--ginkgo.focus=\[Feature:Performance\] --minStartupPods=8
       - --timeout=300m
-
-# Kubernetes e2e correctness tests against EKS 1.10 build.
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-aws/eks/eks-periodics.yaml]: The removal of this might raise eyebrows, but I get it, this isn't something we as a community support.

File: config/jobs/kubernetes/sig-node/node-kubelet.yaml [hunk 13]
```diff
@@ -360,27 +360,27 @@ periodics:
...
       - --deployment=node
       - --gcp-project=k8s-jkns-ci-node-e2e
       - --gcp-zone=us-west1-b
-      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config-1-10.yaml
+      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config-1-11.yaml
       - --node-test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
       - --node-tests=true
       - --provider=gce
-      - --test_args=--nodes=8 --skip="\[Flaky\]|\[Serial\]"
+      - --test_args=--nodes=8 --focus="\[NodeConformance\]" --skip="\[Flaky\]|\[Serial\]"
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-node/node-kubelet.yaml]: Not sure I follow why the change in focus here

File: config/jobs/kubernetes/sig-release/kubernetes-builds.yaml [hunk 14]
```diff
@@ -137,23 +105,23 @@ periodics:
...
-      - --repo=k8s.io/kubernetes=release-1.12
+      - --repo=k8s.io/kubernetes=release-1.13
       - --repo=k8s.io/release
       - --root=/go/src
       - --timeout=180
       - --scenario=kubernetes_build
       - --
       - --allow-dup
-      - --extra-publish-file=k8s-stable1
+      - --extra-publish-file=k8s-beta
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-release/kubernetes-builds.yaml]: job name references `stable1` but this is referencing `beta`, I'm also unclear on why the interval bump

File: config/jobs/kubernetes/sig-release/kubernetes-builds.yaml [hunk 15]
```diff
@@ -165,23 +133,23 @@ periodics:
...
-      - --repo=k8s.io/kubernetes=release-1.11
+      - --repo=k8s.io/kubernetes=release-1.12
       - --repo=k8s.io/release
       - --root=/go/src
       - --timeout=180
       - --scenario=kubernetes_build
       - --
       - --allow-dup
-      - --extra-publish-file=k8s-stable2
+      - --extra-publish-file=k8s-stable1
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-release/kubernetes-builds.yaml]: job name references `stable2` but this is referencing `stable1`, I'm also unclear on why the interval bump

File: config/jobs/kubernetes/sig-release/kubernetes-builds.yaml [hunk 16]
```diff
@@ -193,23 +161,23 @@ periodics:
...
-      - --repo=k8s.io/kubernetes=release-1.10
+      - --repo=k8s.io/kubernetes=release-1.11
       - --repo=k8s.io/release
       - --root=/go/src
       - --timeout=180
       - --scenario=kubernetes_build
       - --
       - --allow-dup
-      - --extra-publish-file=k8s-stable3
+      - --extra-publish-file=k8s-stable2
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-release/kubernetes-builds.yaml]: same here, job name is `stable3` vs the use of `stable2` here, and an interval bump I don't understand

File: config/jobs/kubernetes/sig-testing/integration.yaml [hunk 17]
```diff
@@ -82,7 +82,7 @@ periodics:
         requests:
           cpu: 4
 
-# manual-release-bump-required
+  # manual-release-bump-required
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-testing/integration.yaml]: spacing nit

File: config/jobs/kubernetes/sig-testing/integration.yaml [hunk 18]
```diff
@@ -131,7 +131,7 @@ periodics:
           cpu: 4
 
 # manual-release-bump-required
-- interval: 6h
+- interval: 2h
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-testing/integration.yaml]: unsure why the lowered interval here

File: config/jobs/kubernetes/sig-testing/integration.yaml [hunk 19]
```diff
@@ -155,7 +155,7 @@ periodics:
           cpu: 4
 
 # manual-release-bump-required
-- interval: 24h
+- interval: 6h
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-testing/integration.yaml]: unsure why the lowered interval here

File: config/jobs/kubernetes/sig-testing/verify.yaml [hunk 20]
```diff
@@ -116,29 +153,45 @@ periodics:
         requests:
           cpu: 4
 
-# manual-release-bump-required
+  # manual-release-bump-required
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-testing/verify.yaml]: whitespace nit

File: config/jobs/kubernetes/sig-testing/verify.yaml [hunk 21]
```diff
@@ -165,20 +218,20 @@ periodics:
           cpu: 4
 
 # manual-release-bump-required
-- interval: 6h
+- interval: 2h
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-testing/verify.yaml]: unsure why the lower interval for the stable2 job

File: config/jobs/kubernetes/sig-testing/verify.yaml [hunk 22]
```diff
@@ -189,20 +242,20 @@ periodics:
           cpu: 4
 
 # manual-release-bump-required
-- interval: 24h
+- interval: 6h
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-testing/verify.yaml]: unsure why the lower interval

File: jobs/e2e_node/image-config-1-14.yaml [hunk 23]
```diff
@@ -5,8 +5,8 @@ images:
   ubuntu:
     image: ubuntu-gke-1604-xenial-v20170420-1 # docker 1.12.6
     project: ubuntu-os-gke-cloud
-  coreos-alpha:
-    image: coreos-alpha-2016-0-0-v20190108 # docker 18.06.1
+  coreos-beta:
+    image: coreos-beta-1883-1-0-v20180911 # docker 18.06.1-ce
```
Comment (spiffxp) [inline jobs/e2e_node/image-config-1-14.yaml]: unclear why this change is here

File: testgrid/config.yaml [hunk 24]
```diff
@@ -7849,3 +7753,7 @@ dashboard_groups:
 - name: redhat
   dashboard_names:
   - redhat-openshift-release-blocking
+
+
+
+
```
Comment (spiffxp) [inline testgrid/config.yaml]: whitespace nit

File: config/jobs/kubernetes/kops/kops-config.yaml [hunk 25]
```diff
@@ -146,7 +146,7 @@ presubmits:
       preset-e2e-platform-aws: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-master
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-master
```
[the remark this answers is not in the document]
Comment (krzyzacy) [reply config/jobs/kubernetes/kops/kops-config.yaml]: so this is for https://github.com/kubernetes/kops and looks like they don't have up-to-date release branches - I'll just clean up all branch-specific jobs for this repo

File: config/jobs/kubernetes/sig-gcp/gce-conformance.yaml (hunk 8 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-gcp/gce-conformance.yaml]: I don't think so, link to PR?

File: config/jobs/kubernetes-security/generated-security-jobs.yaml (hunk 4 above)
Comment (imkin) [reply config/jobs/kubernetes-security/generated-security-jobs.yaml]: may be we should add description field into the job configs. This will help generate description too.

File: config/jobs/kubernetes/sig-release/kubernetes-builds.yaml (hunk 11 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-release/kubernetes-builds.yaml]: We version kubekins per release because different release want to use different go/bazel versions. This job doesn't require any of those so version shouldn't matter.

File: config/jobs/kubernetes/kops/kops-config.yaml (hunk 9 above)
Comment (imkin) [reply config/jobs/kubernetes/kops/kops-config.yaml]: @krzyzacy is fixing this.

Comment (krzyzacy) [reply config/jobs/kubernetes/kops/kops-config.yaml]: so - this is a job for kubernetes/kops repo - if you look at https://github.com/kubernetes/kops, they don't have up-to-date release branches, so I'm going to drop all the release branch checks. (the set up is incorrect)

File: config/jobs/kubernetes/sig-aws/eks/eks-periodics.yaml (hunk 12 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-aws/eks/eks-periodics.yaml]: I think eks can still run 1.10 CI (if they care) in their prow instance, instead of community prow instance :-) cc @gyuho @shyamjvs

File: jobs/e2e_node/image-config-1-14.yaml (hunk 23 above)
Comment (krzyzacy) [reply jobs/e2e_node/image-config-1-14.yaml]: this is a bad github diff - we are deleting the 1.10 config and adding 1.14 config, github treats this as a rename.

File: config/jobs/kubernetes/sig-testing/verify.yaml (hunk 21 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-testing/verify.yaml]: :man_shrugging:

File: jobs/e2e_node/image-config-1-14.yaml [hunk 26]
```diff
@@ -5,8 +5,8 @@ images:
   ubuntu:
     image: ubuntu-gke-1604-xenial-v20170420-1 # docker 1.12.6
```
Comment (imkin) [inline jobs/e2e_node/image-config-1-14.yaml]: Where do we use this image in 1.14 release (noting that the docker version is unsupported and deprecated).

Comment (krzyzacy) [reply jobs/e2e_node/image-config-1-14.yaml]: no idea - cc @kubernetes/sig-node-pr-reviews

File: config/jobs/kubernetes/sig-testing/verify.yaml (hunk 22 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-testing/verify.yaml]: blame amit :-) done

File: config/jobs/kubernetes/sig-testing/verify.yaml (hunk 21 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-testing/verify.yaml]: done

File: config/jobs/kubernetes/sig-node/node-kubelet.yaml (hunk 13 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-node/node-kubelet.yaml]: it flows down from the previous stable2 job

File: config/jobs/kubernetes/sig-release/kubernetes-builds.yaml (hunk 14 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-release/kubernetes-builds.yaml]: good catch... here are some bad changes going on :joy:

Comment (krzyzacy) [reply config/jobs/kubernetes/sig-release/kubernetes-builds.yaml]: I think @amwat changed the job name and added a new one on the top? fixing..

Comment (krzyzacy) [reply config/jobs/kubernetes/sig-release/kubernetes-builds.yaml]: done

File: config/jobs/kubernetes/sig-release/kubernetes-builds.yaml (hunk 15 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-release/kubernetes-builds.yaml]: done

File: config/jobs/kubernetes/sig-release/kubernetes-builds.yaml (hunk 16 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-release/kubernetes-builds.yaml]: done

File: config/jobs/kubernetes/sig-testing/integration.yaml (hunk 17 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-testing/integration.yaml]: done

File: config/jobs/kubernetes/sig-testing/integration.yaml (hunk 19 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-testing/integration.yaml]: fixed

File: config/jobs/kubernetes/sig-testing/integration.yaml (hunk 18 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-testing/integration.yaml]: fixed

File: testgrid/config.yaml (hunk 24 above)
Comment (krzyzacy) [reply testgrid/config.yaml]: fixed

File: config/jobs/kubernetes/kops/kops-config.yaml (hunk 10 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/kops/kops-config.yaml]: job removed

File: config/jobs/kubernetes/kops/kops-config.yaml (hunk 9 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/kops/kops-config.yaml]: job removed

File: config/jobs/kubernetes/kops/kops-config.yaml (hunk 25 above)
[the remark this answers is not in the document]
Comment (krzyzacy) [reply config/jobs/kubernetes/kops/kops-config.yaml]: job removed

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml (hunk 6 above)
Comment (krzyzacy) [reply config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml]: will let repo owner figure out when their repo want to move forward, instead of flip version and break their code base

File: config/jobs/kubernetes-security/generated-security-jobs.yaml (hunk 4 above)
Comment (krzyzacy) [reply config/jobs/kubernetes-security/generated-security-jobs.yaml]: there's
```
# Autogenerated by genjobs.go, do NOT edit!
# see genjobs.go, which you can run with hack/update-config.sh
```
at the top... maybe we don't even need to check-in this config? /shrug

feel free to open an issue/PR if you think it bothers you.

File: jobs/e2e_node/image-config-1-14.yaml (hunk 26 above)
Comment (imkin) [reply jobs/e2e_node/image-config-1-14.yaml]: May be will also ping sig-node slack after this goes through.

**Comment by krzyzacy** [comment]
lol - `# manual-release-bump-required` is not very error prune, I think all outstanding comments are addressed

File: config/jobs/kubernetes/sig-gcp/gce-conformance.yaml (hunk 8 above)
Comment (imkin) [reply config/jobs/kubernetes/sig-gcp/gce-conformance.yaml]: Then this seems to be wrong. https://github.com/kubernetes/test-infra/blob/cfdefe0938f4b3ab0c962b1fe75211b1f2e741e4/config/jobs/kubernetes/sig-gcp/gce-conformance.yaml#L87

File: jobs/e2e_node/image-config-1-14.yaml (hunk 26 above)
Comment (yujuhong) [reply jobs/e2e_node/image-config-1-14.yaml]: This should be updated to a newer version, see #10820
/cc @yguo0905

File: config/jobs/kubernetes/sig-gcp/gce-conformance.yaml (hunk 8 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-gcp/gce-conformance.yaml]: why?

Comment (krzyzacy) [reply config/jobs/kubernetes/sig-gcp/gce-conformance.yaml]: fixed the extract line.

stable-1.14 will not work until 1.14 is released, so punt till then.

Comment (imkin) [reply config/jobs/kubernetes/sig-gcp/gce-conformance.yaml]: line 88 says "latest" and line 99 says "stable".

**Comment by imkin** [comment]
I am lgtm. 

Ping @spiffxp to see if he thinks this can now be merged.

**Comment by spiffxp** [comment]
/lgtm
thanks for working through all of this

**Comment by krzyzacy** [comment]
looks like the testgrid dashboard will need a bit more work, will check back the status in a bit.

**Comment by amwat** [comment]
Thanks a lot for taking care of this !

File: config/jobs/kubernetes/sig-testing/verify.yaml (hunk 22 above)
Comment (amwat) [reply config/jobs/kubernetes/sig-testing/verify.yaml]: /shrug this is a combination of both a bad git diff plus not knowing why we run the various stableX jobs on different intervals.
